source/account_structure/index.html.md.erb

```diff
@@ -1,92 +1,68 @@
+### Send your users to different GOV.UK Pay services
 
-### Type 1 (GOV.UK Pay separate, PSP separate)
+Send your users to a different GOV.UK Pay service for each of your services if you need to do any of the following:
 
-<%= image_tag "/images/accountstructure_1.svg", { :alt => '' } %>
+- give each service its own name on the GOV.UK Pay payment pages
+- report on each service's payments separately
+- send each service's payments to different bank accounts
+- give your staff members access to each service separately in the GOV.UK Pay admin tool
 
-The diagram show two services, ‘Parking permits’ and ‘Parking fines’, that are considered separate services by
-both GOV.UK Pay and the PSP.
+To set up your services like this, [create a new GOV.UK Pay service](https://selfservice.payments.service.gov.uk/my-services/create) for each of your online services.
 
-If you use this account structure, you can:
+GOV.UK Pay services are free, and there's no limit to how many you can create.
 
-- give access for each service to different staff members
-- optionally use different service names on the GOV.UK Pay payment pages
-- send each service's payments to a different bank account
-- report separately on each of your services
+To change a service's name, go to the [**My services** page in the GOV.UK Pay admin tool](https://selfservice.payments.service.gov.uk/my-services).
 
-### Type 2 (GOV.UK Pay combined, PSP combined)
+### Send your users to a combined GOV.UK Pay service
 
-<%= image_tag "/images/accountstructure_2.svg", { :alt => '' } %>
+Send your users to a combined GOV.UK Pay service if you need to:
 
-The diagram shows two services, ‘Parking permits’ and ‘Parking fines’, that are considered a single combined
-service by GOV.UK Pay and the PSP.
+- report on all the services' payments together
+- change settings easily across all the services, for example which card types you accept
 
-You may want to use this configuration if you want to simplify the administration of multiple services in GOV.UK Pay. For example, if you do not need reporting at an individual service level.
+You will not be able to give the services different names or send payments to different bank accounts.
 
-The transactions from both services will appear in the same transactions list and CSV file. You may find it useful to use reference numbers to separate payments clearly, for example “parking_fine_1234” and “parking_permit_1234”.
+You may need to use a different [reference number](/making_payments/#reference) or add [custom metadata](/custom_metadata/#add-custom-metadata) for each of your services, so you can tell which service each payment comes from.
 
-In this configuration, payments to services must go to the same bank account because
-they are a single service in the PSP. It is not possible to support multiple PSP accounts in a single GOV.UK Pay service.
+To set up a group of your services like this, use the same API key or payment link for all the services in the group.
```

Comment:
you can set up multiple payment links in one service. this means you can change the settings in each one (like reference number field name, description, reporting columns). but they will all be on the same CSV download within that service. a user would have to get all the data for that service and then filter by description / reporting column to separate out different transaction types.

Reply:
As mentioned in another comment -  I've added using different API keys / payments links to the list of ways you can distinguish between services, but may have the details wrong?